packages/react-components/package.json

```diff
@@ -39,12 +39,14 @@
     "@fluentui/babel-make-styles": "9.0.0-beta.4"
   },
   "dependencies": {
+    "@fluentui/react": "8.49.0",
```

Comment:
This shouldn't be added to react-components. You need to find another way to achieve the same thing without v8 components.

Reply:
Yeah I think I can use the react-input and choicegroup components

Reply:
react-radio isn't ready yet, so you'll have to just use the built-in `<input type="radio">`. Don't worry about about styling it to look nice for now, and we can replace it with Radio once that's ready.

Reply:
Instead of SearchBox you can do something like `<Input type="search" contentBefore={<SearchIcon/>} />` (plus a label or aria-label for the Input) and don't worry about having a clear button.